database/seedRecordsPG.js

```diff
@@ -79,16 +73,20 @@ var dataGenerator = (id) => {
       "http://placeimg.com/640/480/nature"
     ];
     for (i = 0; i < count; i++) {
-      imagesArr.push(links[i]);
+      if (i === count-1) {
+        imagesArr += links[i];
+      } else {
+        imagesArr += links[i] + "-"
+      }
     }
     return imagesArr;
   }
   const images = imagesGen();
-  return `${id},${description},${articleNumber},${details},${materials},${sustainability},${width},${height},${length},${weight},${packages},${shortDesc},${threadCount},${pillowcaseQuantity},${duvetCoverLength},${duvetCoverWidth},${pillowcaseLength},${pillowcaseWidth},${fitting}, ${images}\n`;
+  return `${id},${name},${description},${articleNumber},${details},${materials},${sustainability},${width},${height},${length},${weight},${packages},${shortDesc},${threadCount},${pillowcaseQuantity},${duvetCoverLength},${duvetCoverWidth},${pillowcaseLength},${pillowcaseWidth},${fitting}, ${images}\n`;
```

Comment:
curious how this is working with your postgres serial id and the id generated to your csv in this function--> is one overriding the other when you copy to the table? I'm not sure on how serial id works, but my impression was that it was like auto increment in mysql-- is it being replaced by the csv id or vice versa? or neither and I'm mixed up lol.

Reply:
Funny enough, I had to do this because I couldn't leave "serial" blank. You know how in MySQL you could leave "auto increment" it would then, well, auto increment? lol. Well I wasn't able to do that when copying the data over from CSV to the database, but please let me know if you find a different solution.